narayana/pom.xml

```diff
@@ -98,6 +98,13 @@
 		        <type>pom</type>
 		        <scope>import</scope>
 		    </dependency>
+            <dependency>
+                <groupId>org.jboss.narayana</groupId>
+                <artifactId>narayana-bom-test</artifactId>
+                <version>6.0.1.Final-SNAPSHOT</version>
```

Comment:
can we use `<version>${version.org.jboss.narayana}</version>` here?

Reply:
I double checked and actually we cannot because the version.org.jboss.narayana is not set in narayana pom, but only in comparison pom. And at the moment they don't have a common parent.

Reply:
@jmfinelli I've just moved the 'narayana.version' property so that we could use it to specify the BOMs version too